### PR TITLE
:book: Fix typo in designs template

### PR DESCRIPTION
**Pull request by gabriele-wolfox**
This patch fixes a typo inside designs template.
"Implementeble" should be replaced by "Implementable".

Signed-off-by: gabriele-wolfox <gabriele.wolfox@gmail.com>

### CODE REVIEW

**Comment by gabriele-wolfox** [comment]
@mateusoliveira43 moving the PR here in this branch.
Thanks for the help, it is my first PR here.

**Comment by mateusoliveira43** [comment]
@gabriele-wolfox no problem :smile: 

You can check Contributing docs for more info https://github.com/kubernetes-sigs/kubebuilder/blob/master/CONTRIBUTING.md

I think there is an extra commit changing a dependency version, that should be removed

**Comment by gabriele-wolfox** [comment]
> I think there is an extra commit changing a dependency version, that should be removed

Yes, I am going to remove it tomorrow

File: docs/book/src/component-config-tutorial/testdata/project/go.mod [hunk 1]
```diff
@@ -5,7 +5,7 @@ go 1.20
 require (
 	k8s.io/apimachinery v0.28.1
 	k8s.io/client-go v0.28.1
-	sigs.k8s.io/controller-runtime v0.16.1
+	sigs.k8s.io/controller-runtime v0.16.2
```
Comment (camilamacedo86) [inline docs/book/src/component-config-tutorial/testdata/project/go.mod:8]: You need to revert the changes in the code if the goal is just the doc.

Comment (gabriele-wolfox) [reply docs/book/src/component-config-tutorial/testdata/project/go.mod]: Yeah, the goal is just the doc. 
Removed, and thanks.

**Review by camilamacedo86 — APPROVED** [review]
/lgtm
/approved
